promql/parser/printer.go

```diff
@@ -229,7 +229,6 @@ func (node *Call) ShortString() string {
 }
 
 func (node *MatrixSelector) atOffset() (string, string) {
-	// Copy the Vector selector before changing the offset
```

Comment:
Removed this because the line below doesn't copy, it's still a pointer (and it's not modified, so it's fine).